### DISCUSSION TITLE
Feature Request: Silence Success messages

### DISCUSSION THREAD

**Discussion by kmatt** [duckdb/duckdb#14005] [Ideas]
`DuckDB v1.1.0 fa5c2fe15f`

Why do you want this feature?

When piping query results from the CLI to `/dev/stdout` (Linux), the result of prior statements appears in the piped data, resulting in extra filtering being needed:

```sql
-- test.sql
CREATE SECRET ... ;
COPY (SELECT ... FROM 'azure://...') WITH (header false);
```

If piped into a process with `duckdb -csv -noheader < test.sql | awk '{print}'` includes the `true` value of the "Success" column emitted by the `CREATE SECRET` command. Being able to silence those "Success" messages would keep the data stream clean. 

The `-csv -noheader` CLI parameters are to avoid the extra lines from the formatting of the CREATE SECRET message, not the COPY statement.

This may also be my misunderstanding how Bash redirection works, because if I run instead `duckdb < test.sql > out.csv` the resulting file only has the CSV data. But so far, I have not been able to avoid sending the `true` value of the "Success" column to Awk / Less / Vim etc., including: `duckdb -csv -noheader < zscore.sql > >(mawk '{print}')`

**Comment by davetapley** [comment]
I think it would be a useful feature too.

As a workaround for now you can `.output /dev/null` at the top of `test.sql` then `.output` to switch to back to `stdout`.
